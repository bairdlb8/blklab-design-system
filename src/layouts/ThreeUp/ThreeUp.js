import React from 'react'
import PropTypes from 'prop-types'

import './ThreeUp.scss'

const ThreeUp = ({className=''}) => {
  return <div>ThreeUp</div>
}

ThreeUp.propTypes = {}

export default ThreeUp
