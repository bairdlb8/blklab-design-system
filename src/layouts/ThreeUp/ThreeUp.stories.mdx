import { Meta, Story, Props, Preview } from '@storybook/addon-docs/blocks'

import ThreeUp from './ThreeUp'

<Meta title="Layouts / ThreeUp" component={ ThreeUp } />

# ThreeUp

## Add some text here

export const Template = (args) => {
  return <ThreeUp {...args} />
}

<Preview withToolbar>
  <Story name="Primary" args={ {} }>
    {Template.bind({})}
  </Story>
</Preview>

<Props of={ ThreeUp } />
