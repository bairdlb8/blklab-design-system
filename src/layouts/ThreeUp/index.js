import ThreeUp from './ThreeUp'
export default ThreeUp
