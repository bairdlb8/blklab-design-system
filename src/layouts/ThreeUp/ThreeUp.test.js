import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { axe, toHaveNoViolations } from 'jest-axe'
import ThreeUp from './ThreeUp'

expect.extend(toHaveNoViolations)

describe('<ThreeUp />', () => {
  describe('accessibility', () => {
    it('should not fail any accessibility tests', async () => {
      const { container } = render(<ThreeUp />)
      expect(await axe(container)).toHaveNoViolations()
    })
  })

  it('should take a snapshot', () => {
    const { asFragment } = render(<ThreeUp />)

    expect(asFragment(<ThreeUp />)).toMatchSnapshot()
  })
})
