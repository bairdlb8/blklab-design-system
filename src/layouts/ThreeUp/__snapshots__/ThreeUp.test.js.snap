// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ThreeUp /> should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    ThreeUp
  </div>
</DocumentFragment>
`;
